text_extensions_for_pandas/io/test_tokenization.py

```diff
@@ -76,3 +79,28 @@ def test_make_bert_tokens(self):
         char_span_text = [s.covered_text for s in char_span]
         token_span_text = [s.covered_text for s in token_span]
         self.assertListEqual(char_span_text, token_span_text)
+
+    def test_seq_to_windows(self):
+        for seqlen in range(1, 20):
+            seq = np.arange(1, seqlen)
+            seq_after = windows_to_seq(seq, seq_to_windows(seq, 2, 3)["input_ids"], 2, 3)
+            if np.any(seq != seq_after):
+                raise ValueError("Before: {seq}; After: {seq_after}")
+
+        for seqlen in range(200, 400):
+            # print(f"seqlen: {seqlen}")
```

Comment:
leftover print statements?

Reply:
Good catch! I've removed these statements in dbb7da9